docs/basics/integrations/deque.md

```diff
@@ -0,0 +1,137 @@
+---
+id: deque
+title: "Deque axe™ Integration"
+sidebar_label: "Axe Deque"
+keywords:
+- accessibility-testing
+- accessibility
+- automated-testing
+- how-to
+---
+
+<p><small><span class="highlight beta">BETA</span></small></p>
+
+[Deque's axe™](https://www.deque.com/axe/) is one of the world's leading digital accessibility toolkits. The Sauce Labs integration allows you to run your accessibility tests, using axe™, on our platform.
+
+## What You'll Need
+* [A Sauce Labs Account](https://saucelabs.com/sign-up)
+* [A Deque Account (Optional)](https://axe.deque.com/plans)
+* [Install Git](https://git-scm.com/downloads)
+* [Install Node/NPM](https://nodejs.org/en/download/)
+
+## Project Setup
+
+Clone or download the [example project](https://github.com/saucelabs/axe-wdio):
+
+```bash
+git clone https://github.com/saucelabs/axe-wdio.git
+```
+From the main project directory, change the following lines in `wdio.saucelabs.conf.js`:
+
+```js {10-12}
+const {config} = require('./wdio.conf');
+const defaultBrowserSauceOptions = {
+    build: `Best Practices: Sauce Labs Desktop Web build-${new Date().getTime()}`,
+    screenResolution: '1600x1200',
+};
+
+// =====================
+// Sauce specific config
+// =====================
+config.user = "yourUserName";
+config.key = "yourAccessKey";
+config.region = "yourRegion -> us / eu";
+
+// ============
+// Capabilities
+// ============
+config.capabilities = [
+    /**
+     * Desktop browsers
+     */
+    {
+        browserName: 'googlechrome',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    {
+        browserName: 'firefox',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    // {
+    //     browserName: 'internet explorer',
+    //     platformName: 'Windows 8.1',
+    //     browserVersion: 'latest',
+    //     'sauce:options': {
+    //         ...defaultBrowserSauceOptions,
+    //     },
+    // },
+    {
+        browserName: 'MicrosoftEdge',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+    // Safari 13
+    {
+        browserName: 'safari',
+        platformName: 'macOS 10.15',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+        },
+    },
+];
+
+// ========
+// Services
+// ========
+config.services = config.services.concat('sauce');
+
+exports.config = config;
+
+```
+
+### Sauce Labs Regions
```

Comment:
This heading is too restrictive, since the warning below applies to all three configs, not just region. Maybe something like `###  Sauce Labs Configs`?

Reply:
👍